Doc/library/sqlite3.rst

```diff
@@ -394,6 +394,20 @@ Connection Objects
       supplied, this must be a callable returning an instance of :class:`Cursor`
       or its subclasses.
 
+   .. method:: blobopen(table, column, row, /, *, readonly=False, name="main")
+
+      Open a :class:`Blob` handle to the :abbr:`BLOB (Binary Large OBject)`
+      located in row *row*, column *column*, table *table* of database *name*.
+      When *readonly* is :const:`True` the blob is opened without write
+      permissions.
```

Comment:
What is `row`? Is it rowid? What about without-rowid tables?

The documentation and the function signature mentiont row, column, table in a different order.

Reply:
> What is row? Is it rowid?

Yes.

> What about without-rowid tables?

Blob open will fail with an exception (probably `OperationalError`, but I didn't check)

> The documentation and the function signature mentiont row, column, table in a different order.

I'll fix that in a separate PR. Thanks.

Reply:
See GH-91561